syft/frameworks/torch/mpc/falcon/falcon_helper.py

```diff
@@ -89,8 +89,8 @@ def select_share(
         c_2 = c.share(*players, protocol="falcon", field=2)
         c_L = c.share(*players, protocol="falcon", field=ring_size)
 
-        xor_b_c = FalconHelper.xor(b, c_2).reconstruct()
-        d = c_L * (1 - 2 * xor_b_c) + xor_b_c
+        xor_b_c = FalconHelper.xor(b, c_2.child).reconstruct()
```

Comment:
Q: What happens here if you don't select the child? (it should be a simple Wrapper, right?)

Reply:
the xor function calls the __eq__ of native, which as far as i know calls the overload 
```
@wraps(getattr(tensor_type, method_name))
    def overloaded_native_method()
```
which removes the wrapper and tried to use the eq operation on RST (which has not been implemented yet) 
which ultimately throws up the error 
`AttributeError: 'ReplicatedSharingTensor' object has no attribute 'eq' `

Reply:
What is happening here...I think..is that when ```other``` get ```wrapped``` ```isinstance(other, LongTensor)``` returns true and we do the ```eq```. I will think of a way to avoid this and will ping you on Slack :D

Reply:
okay :)

Reply:
You should be able to pass ```c_2```, right?